src/components/Settings/SlippageTolerance.vue

```diff
@@ -165,6 +159,10 @@ export default class SlippageTolerance extends Mixins(mixins.NumberFormatterMixi
   handleSetTransactionDeadline(value: number): void {
     this.setTransactionDeadline(value);
   }
+
+  handleCollapseChange() {
```

Comment:
Could you add `void` return type, please?

Reply:
fixed